platform/darwin/test/MGLVersionNumber.m

```diff
@@ -0,0 +1,15 @@
+#import <Mapbox/Mapbox.h>
+
+#import <XCTest/XCTest.h>
+
+@interface MGLVersionTests : XCTestCase
+
+@end
+
+@implementation MGLVersionTests
+
+- (void)testVersionNumber {
+    XCTAssertEqual(1, MapboxVersionNumber);
+}
```

Comment:
Added a test here to check linkage. This is not a Swift-specific issue; it also happened with Objective-C.